src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -889,6 +889,86 @@ ENC(xtlMWGd0ZK/EzTMyrgPiPw==)
 The encrypted value can then be used as displayed by the `encrypt` command, for example, in *PROACTIVE_HOME/config/scheduler/database.properties*:
 `hibernate.connection.password=ENC(xtlMWGd0ZK/EzTMyrgPiPw==)`
 
+=== Configure Global Variables and Generic Information
+
+ProActive Server allows to configure workflow variables or generic information that will apply to all submitted workflows or to some categories of workflows (e.g. workflows with a given name).
+
+This configuration is available in the file *PROACTIVE_HOME/config/scheduler/global_variables.xml*.
+
+
+The default configuration is the following:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<globalvariables
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns="urn:proactive:globalvariables:3.13" xsi:schemaLocation="urn:proactive:globalvariables:3.13 http://www.activeeon.com/public_content/schemas/proactive/globalvariables/3.13/globalvariables.xsd">
+        <filter>
+            <select>
+                <!-- Xpath expressions are used to filter workflows affected by global variables.
+                Example: to select workflows belonging to the basic-examples bucket
+                <xpath><![CDATA[/job/genericInformation/info[@name='bucketName' and @value='basic-examples']]]></xpath>
+                -->
+                <xpath><![CDATA[.]]></xpath>
+                <!-- more than one xpath expression can be added. In that case, global variables are applied when all xpath expressions match -->
+            </select>
+            <!-- add global variables or generic information below -->
+            <variables>
+            </variables>
+            <genericInformation>
+            </genericInformation>
+        </filter>
+        <!-- Multiple filters can be added, to apply different set of global variables to different workflows -->
+</globalvariables>
+```
+
+This configuration file is composed of `<filter>` sections. Each filter section describes the global variables and generic information that will be added to a certain category of workflows.
+Multiple filter sections allows to add different lists of variables to different types of workflows.
```

Comment:
Shouldn't be "allow" instead of "allows" ?

Reply:
right